src/applications/simple-forms/mock-alternate-header-21-0845/containers/IntroductionPage.jsx

```diff
@@ -56,30 +66,17 @@ class IntroductionPage extends React.Component {
           VA will no longer give out benefit or claim information (except for
           the information VA has already given out based on your permission).
         </p>
-        <SaveInProgressIntro
-          headingLevel={2}
-          alertTitle="Save time and save your work in progress by signing in"
-          prefillEnabled={formConfig.prefillEnabled}
-          messages={formConfig.savedFormMessages}
-          pageList={pageList}
-          startText="Start the Application"
-          verifiedPrefillAlert={
-            <div>
-              <div className="usa-alert usa-alert-info schemaform-sip-alert">
-                <div className="usa-alert-body">
-                  <strong>Note:</strong> Since you’re signed in to your account,
-                  you can save your release authorization in progress and come
-                  back later to finish filling it out.
-                </div>
-              </div>
-              <br />
-            </div>
-          }
-          displayNonVeteranMessaging
-        >
-          Please complete the 21-0845 form to apply for disclosure
-          authorization.
-        </SaveInProgressIntro>
+        <p>
```

Comment:
copied relevant code from `SaveInProgressIntro`, so that only the unauthenticated link shows. This is a workaround for a bug where we get a spinner indefinitely on this page - maybe related to content build no nav or login?  Unsure.  But we don't need authenticated experience for this mock form.